pages/api/posts/publish.js

```diff
@@ -24,13 +29,13 @@ const publishTeam = async (...args) => {
     args[0].author.name
   )}-${getNormalizedDate()}.json`;
 
-  args[0].filePath = `${path}/${args[0].data.team}/${fileName}`;
+  args[0].filePath = `${path}/${args[0].data.team.slug}/${fileName}`;
 
   return await publishData(...args);
 };
```

Comment:
i think async/await is not necessary here as the await is in the return value only